src/backend/access/heap/heapam.c

```diff
@@ -9292,7 +9296,7 @@ heap_xlog_multi_insert(XLogReaderState *record)
 			htup->t_infomask = xlhdr->t_infomask;
 			htup->t_hoff = xlhdr->t_hoff;
 			HeapTupleHeaderSetXmin(htup, XLogRecGetXid(record));
-			HeapTupleHeaderSetCmin(htup, xlhdr->t_cid);
+			HeapTupleHeaderSetCid(htup, xlhdr->t_cid);
```

Comment:
this should not change, cmin should never be a combo id.

Reply:
Thanks for your serveral suggestion. I'll change the code later